tests/e2e/README.md

```diff
@@ -0,0 +1,33 @@
+# InstructLab Pipeline Tests on RHOAI
```

Comment:
can we rename the dir to tests/pipeline/e2e instead to be consistent with the other directories in `tests`  ?